src/components/FeaturePanel/Property.tsx

```diff
@@ -34,12 +34,16 @@ const Value = styled.div`
 `;
 
 const DefaultRenderer = ({ v }) => v;
-const renderers = {
+var renderers = {
   website: WebsiteRenderer,
   phone: PhoneRenderer,
   opening_hours: OpeningHoursRenderer,
 };
 
+renderers["contact:website"] = WebsiteRenderer;
+renderers["contact:phone"] = PhoneRenderer;
+renderers["contact:mobile"] = PhoneRenderer;
```

Comment:
I prefer sth like:

```
const renderers = {
  ...
  'contact:website': WebsiteRenderer,
}
```

Reply:
I had that originally, but missed the other place to add the tags